cmd/skopeo/inspect.go

```diff
@@ -52,6 +54,7 @@ See skopeo(1) section "IMAGE NAMES" for the expected format
 	flags := cmd.Flags()
 	flags.BoolVar(&opts.raw, "raw", false, "output raw manifest or configuration")
 	flags.BoolVar(&opts.config, "config", false, "output configuration")
+	flags.StringVarP(&opts.format, "format", "f", "", "Format the output to a Go template or json")
```

Comment:
This does not really explain how to use the value.

Reply:
We don't explain it in any of our tools, and yet people seem to understand it.

Reply:
There is an example in the man page.

Reply:
To be specific, at least please put the JSON value in quotes (`or "json"`) to make it clear that it is a literal to be used, not a incorrectly-lowercased reference to a generic concept.

Reply:
… actually, the `json` value was completely removed? So, this description needs to be updated but it’s easy now.

Reply:
Removed json.